src/templates/template-vision-dcgan/main.py

```diff
@@ -126,14 +129,14 @@ def run(local_rank: int, config: Any):
     @trainer.on(Events.EPOCH_COMPLETED)
     def save_fake_example(engine):
         fake = model_g(fixed_noise)
-        path = config.output_dir / FAKE_IMG_FNAME.format(engine.state.epoch)
+        path = config.output_dir + "/" + FAKE_IMG_FNAME.format(engine.state.epoch)
```

Comment:
let's revert this and similar updates. We are using Pathlib where we can use `/` operator